ubiquibot-config-default.json

```diff
@@ -8,6 +8,8 @@
   "disable-analytics": false,
   "comment-incentives": false,
   "register-wallet-with-verification": false,
+  "openai-api-key": null,
+  "openai-max-tokens": 8000,
```

Comment:
This is still 8000

Reply:
1/ Why put the null value in default.json here?
2/ @pavlovcik What's your recommendation value? If you put it in the comment, I can directly push to speed up the merge.